qcodes/instrument_drivers/AlazarTech/ATS9870.py

```diff
@@ -150,6 +150,23 @@ def __init__(self, name, **kwargs):
                            unit='10 us',
                            value=0,
                            vals=validators.Ints(min_value=0))
+        self.add_parameter(name='aux_io_mode',
+                           parameter_class=AlazarParameter,
+                           label='AUX I/O Mode',
+                           unit=None,
+                           value='AUX_IN_AUXILIARY',
+                           byte_to_value_dict={0: 'AUX_OUT_TRIGGER',
+                                               1: 'AUX_IN_TRIGGER_ENABLE',
+                                               13: 'AUX_IN_AUXILIARY'})
```

Comment:
is 13 a valid value? we could not find it in the manual

Reply:
For the 9360 it is but maybe that we have to overwrite this in that class